letsencrypt/rootfs/etc/services.d/lets-encrypt/run

```diff
@@ -4,6 +4,7 @@
 # ==============================================================================
 CERT_DIR=/data/letsencrypt
 WORK_DIR=/data/workdir
+LOGS_DIR=/ssl/letsencrypt-logs
```

Comment:
Apart from what has been said already, I don't think that the `/ssl` directory is appropriate to save logs. IMO it should just contain the certificates and secrets but not runtime logs.